assets/diagram-js.css

```diff
@@ -176,6 +176,20 @@ marker.djs-dragger text {
   pointer-events: none;
 }
 
+/**
+ * hit shape
+ */
+.djs-hit {
+  fill: none;
+  stroke: white;
+  stroke-opacity: 0.0;
+  stroke-width: 15px;
+}
+
+.djs-hit-extended .djs-hit {
```

Comment:
We already apply a `attach-ok` marker / class to elements that are show the attach preview. Could we not make the extended hit shape a generic behavior? I.e. 

```
.djs-element.attach-ok .djs-hit {
  stroke-width: 80px;
}
```

Reply:
I do not see a reason why not all editors should inherit the sticky behavior for attachers.

Reply:
This makes sense.